tests/utils.py

```diff
@@ -41,6 +45,91 @@ def login(flask_client) -> User:
     return user
 
 
+def alias_export(flask_client, target_url):
```

Comment:
We keep this file for small utils re-used across lots of tests. Not only two. Can you move this function to a different file like `alias_test_utils.py` ?

Reply:
done !